app/src/main/res/values-de/strings.xml

```diff
@@ -1,65 +1,57 @@
-    <string name="remove_from_playlist_description">Diese Aktion wird nur alle Dateien der Playlist \"%s\" löschen, nicht die eigentlichen Dateien.</string>
+    <string name="remove_playlist">Entferne Wiedergabeliste</string>
+    <string name="remove_playlist_description">Entfernt werden nur die ausgewählten Wiedergabelisten, nicht die eigentlichen Dateien.</string>
+    <string name="remove_playlist_description_placeholder">Entfernt wird nur die Wiedergabeliste \"%s\", nicht die tatsächlichen Dateien.</string>
+    <string name="remove_from_playlist">Von Wiedergabeliste entfernen</string>
+    <string name="remove_from_playlist_description">Es werden nur die Elemente aus der Wiedergabeliste \"%s\" entfernt, die eigentlichen Dateien werden nicht gelöscht.</string>
     <string name="delete_the_files_too">Dateien auch löschen</string>
     <string name="open_playlist">Playlist öffnen</string>
-    <string name="all_songs_cannot_be_deleted">Playlist \'Alle Songs\' kann nicht gelöscht werden</string>
-    <string name="empty_playlist">Die derzeitige Playlist ist leer</string>
+    <string name="all_songs_cannot_be_deleted">Wiedergabeliste \'Alle Lieder\' kann nicht gelöscht werden</string>
+    <string name="empty_playlist">Derzeitige Wiedergabeliste ist leer</string>
     <string name="fetching_songs">Rufe Songs ab…</string>
-    <string name="all_songs">Alle Songs</string>
-    <string name="create_playlist_from_folder">Neue Playlist aus Ordner erstellen</string>
+    <string name="all_songs">Alle Lieder</string>
+    <string name="create_playlist_from_folder">Erstelle neue Wiedergabeliste aus Ordner</string>
     <string name="folder_contains_no_audio">Der ausgewählte Ordner enthält keine Audiodateien</string>
 
     <!-- Settings -->
     <string name="equalizer">Equalizer</string>
 
     <!-- Strings displayed only on Google Playstore. Optional, but good to have -->
     <!-- Short description has to have less than 80 chars -->
-    <string name="app_short_description">Ein schlichter Musikplayer mit anpassbarem Widget, ganz ohne Werbung.</string>
-    <string name="app_long_description">
-        Ein schlichter Musikplayer, der ganz einfach über die Benachrichtigungsleiste, das Homescreen Widget oder die Tasten an deinem Headset gesteuert werden kann. Du kannst alle Songs nach Titel oder Künstler sortieren und den Equalizer anpassen. Das Filtern nach Künstler oder Album wird noch nicht unterstützt, bleib dran!
-        
-        Die App beinhaltet ein Widget mit anpassbarer Textfarbe, Transparenz und Hintergrundfarbe.
-        
-        Beinhaltet keine Werbung oder unnötige Berechtigungen. Sie ist komplett Open Source, alle verwendeten Farben sind anpassbar.
-
-        Diese App ist nur eine aus einer größeren Serie von schlichten Apps. Der Rest davon findet sich auf http://www.simplemobiletools.com
-    </string>
+    <string name="app_short_description">Ein sauberer Musik-Player mit anpassbarem Widget.</string>
```

Comment:
This would contradict the app name `Schlichter Musik-Player` and the `ganz ohne Werbung.` is a nice addition (and maybe an important one for some people looking for apps) that matches with the rest of all the Simple Apps.